content/blog/2020/06/2020-06-08-hackfest-docs-results.adoc

```diff
@@ -0,0 +1,113 @@
+It assures that documentation changes are reviewed by the same maintainers who review and approve new capabilities.
+
+Hackfest participants submitted pull requests to migrate plugin documentation to GitHub.
+10 plugin pull requests are in progress from the Hackfest.
+5 plugin pull requests from the Hackfest have been already merged and are awaiting the release of the plugin.
+
+== Chuck Norris uses documentation as code
+
+In the spirit of fun and adventure, Oleg Nenashev migrated the "Chuck Norris plugin" to GitHub documentation as code in a link:https://www.youtube.com/watch?v=BaEJ8v7INNQ[live Hackfest presentation May 26, 2020].
+Links to the recording, the plugin migration guide, and the export tool are available from link:https://github.com/jenkinsci/ui-ux-hackfest-2020/tree/master/presentations/04-migrating-plugin-docs["Migrating plugins to documentation-as-code"].
+
+image:/images/post-images/2020-06-08-docs-from-hackfest/chuck-norris-docs-as-code.png[Chuck Norris plugin uses documentation-as-code]
+
+== Documentation updates
+
+Jenkins works with other technologies to solve automation challenges in many different environments.
+We describe those environments in our link:/solutions/["Solution Pages"].
+As part of the Hackfest, we've started a series of improvements to the solution pages.
+
+The link:/solutions/[Docker solutions page] now includes updated videos and a better page layout for easier reading and better navigation.
+Other solution pages will receive similar improvements in the future.
+
+image:/images/post-images/2020-06-08-docs-from-hackfest/jenkins-and-docker.png[Jenkins and Docker]
+
+== System properties
+
+The global configuration of Jenkins can be modified at startup by defining Java properties.
+System properties can change system defaults and can provide compatibility "escape hatches" when a new default configuration might be incompatible with existing installations.
+
+Daniel Beck has improved the navigation and user experience of the system properties page as part of the Hackfest.
+It is now much easier to read and to reference, with embeddable links available with a mouse-over to the right of every property and labels that categorize and classify each property.
+
+image:/images/post-images/2020-06-08-docs-from-hackfest/system-properties.png[System Properties]
+
+== Plugin site improvements
+
+During the Hackfest, Gavin Mogan has continued his efforts to improve the link:https://plugins.jenkins.io/[Jenkins Plugins Site] so that users can easily access plugin changelogs and reported issues.
+Once this pull request is merged, it will greatly improve the experience of those Jenkins users who want to update plugins and look for documentation about what has changed in them and what are the possible issues they might experience.
+
+Example of the incoming UI for the Jira plugin:
+
+image:/images/post-images/2020-06-08-docs-from-hackfest/plugins-site.png[Plugins Site]
```

Comment:
UI has changed a little bit since then as it looked a bit weird having all the changelogs in a card style, but not a blocker.